pavics_datacatalog/wps_processes/wps_pavicrawler.py

```diff
@@ -110,9 +89,7 @@ def _handler(self, request, response):
                     wms_alternate_server=wms_alternate_server,
                     target_files=target_files)
         except:
-            logger.error('catalog.pavicrawler raised an Exception',
-                         exc_info=True)
-            update_result = json.dumps({'Exception': 'check log file'})
+            raise Exception(traceback.format_exc())
```

Comment:
ah ouin, il y a un call qui log le "solr_json_output", et un commentaire plus loin de # Should have a logging mechanism for this..., bref rien de valeur en ce moment, mais ça serait à améliorer... on peut remettre ça à plus tard?